### PR TITLE
Update changelog for v7.0.0-beta2

### PR DESCRIPTION
**Pull request by marefr**
Did some updates of PR titles in the PR's before generating. Anything missing?

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
LGTM

File: CHANGELOG.md
Comment (dprokop) [inline CHANGELOG.md:71]:
```suggestion
- Scripted dashboards are now deprecated. The feature is not removed but will be in a future release. We hope to address the underlying requirement of dynamic dashboards in a different way. [#24059](https://github.com/grafana/grafana/issues/24059)
```

File: CHANGELOG.md
Comment (dprokop) [inline CHANGELOG.md:77]:
```suggestion
* **Login Page**: New design. [#23892](https://github.com/grafana/grafana/pull/23892), [@torkelo](https://github.com/torkelo)
```

File: CHANGELOG.md
Comment (dprokop) [inline CHANGELOG.md:85]:
```suggestion
* **Transformations**: Improve time series support. [#23978](https://github.com/grafana/grafana/pull/23978), [@ryantxu](https://github.com/ryantxu)
```

File: CHANGELOG.md
Comment (dprokop) [inline CHANGELOG.md:104]:
```suggestion
* **Panel inspect**: hides Query tab for plugins without Query ability. [#24216](https://github.com/grafana/grafana/pull/24216), [@hugohaggmark](https://github.com/hugohaggmark)
```

**Review by dprokop — APPROVED** [review]

**Review by aknuds1 — APPROVED** [review]
LGTM once dprokop's suggestion is merged :)

**Review by aocenas — APPROVED** [review]

**Review by xlson — COMMENTED** [review]
There is one Enterprise PR that should go in here as well. 

`Reporting: update report schedule as soon as a reported is updated (Enterprise)`

**Comment by marefr** [comment]
@xlson I'll fix :+1:

**Review by daniellee — APPROVED** [review]
LGTM

**Comment by aknuds1** [comment]
@marefr There's now a merge conflict :/

**Pull request merged**
